lib/active_fedora/solr_indexing_strategy.rb

```diff
@@ -0,0 +1,26 @@
+module ActiveFedora
+  ##
+  # An application strategy to also apply solr indexes for each property.
+  # @note If how a field is indexed changes based on property, this would be a
+  #   good place to define that.
+  class SolrIndexingStrategy
```

Comment:
Sorry, but can we change the name one more time. How about `SchemaIndexingStrategy` that makes it clear that it's for Schemas and abstracts out the indexing technology.